packages/apps/plugins/plugin-settings/src/index.ts

```diff
@@ -0,0 +1,13 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { SettingsPlugin } from './SettingsPlugin';
+
+export default SettingsPlugin;
+
+export * from './SettingsPlugin';
+
+// TODO(wittjosiah): Remove.
+//  Settings should be exposed from plugins as state and intents rather than components.
+export * from './components';
```

Comment:
ok to remove this now and expose the dialog via the plugin's surface?

Reply:
That might work, we haven't used surfaces with children a ton but theoretically should work. Not super motivated to make that change since it'd go away once we generate settings UI from data but not opposed to it either.